project/AkkaBuild.scala

```diff
@@ -125,6 +125,19 @@ object AkkaBuild extends Build {
     dependencies = Seq(cluster % "compile->compile;test->test;multi-jvm->multi-jvm", slf4j % "test->compile")
   ) configs (MultiJvm)
 
+  lazy val clusterTools = Project(
+    id = "akka-cluster-tools",
+    base = file("akka-cluster-tools"),
+    dependencies = Seq(cluster % "compile->compile;test->test;multi-jvm->multi-jvm")
+  ) configs (MultiJvm)
+  
+  lazy val clusterSharding = Project(
+    id = "akka-cluster-sharding",
+    base = file("akka-cluster-sharding"),
+    dependencies = Seq(cluster % "compile->compile;test->test;multi-jvm->multi-jvm", 
+        persistence % "compile;test->provided", clusterTools)
```

Comment:
I would be happy to lear what this actually means in sbt terms. For me it reads "take the test dependencies in akka-persistence and make them provided here", but that is not what it is doing.